lib/resource-loader.ts

```diff
@@ -20,8 +21,12 @@ export class ResourceLoader implements IResourceLoader {
 		return this.$fs.readJson(this.resolvePath(resourcePath));
 	}
 
-	public get appResourcesDir(): string {
-		return this.resolvePath("App_Resources");
+	public get cordovaAppResourcesDir(): string {
```

Comment:
Or maybe we don't need separate functions for `cordova` and `nativescript` framework, but one function that get `frameworkName` as parameter and just ` return path.join(this.resolve(frameworkName), this.$projectConstants.APP_RESOURCES_DIR_NAME` . In this case it is ok to be here in `resource-loader` file.